README.md

```diff
@@ -36,58 +46,89 @@ You can set this up on a Docker droplet from [Digital Ocean](https://www.digital
 This [`SETUP.md`](./SETUP.md) explains the Application Server Installation and its setup. [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) are used to make the installation and
 setup easier.
 
-This dashboard uses [docker-compose](https://docs.docker.com/compose/overview/) to set up a group of five primary [docker containers](https://www.docker.com), backed by two auxiliary container:
+This dashboard uses [docker-compose](https://docs.docker.com/compose/overview/) to set up a group of eight primary [docker containers](https://www.docker.com), backed by two auxiliary containers:
 
-1. An instance of [Nginx](https://www.nginx.com/), which proxies the other services, handles access control, gets SSL certificates from [Let's Encrypt](https://letsencrypt.org/), and faces the outside world.
+1. An instance of [Nginx](https://www.nginx.com/), which proxies the other services, handles access control, gets SSL certificates from [Let's Encrypt](https://letsencrypt.org/), and faces the outside world.
```

Comment:
Out of sequence... I've already commented: but we have to look closely at these whitespace changes, because it's likely they're changing to some non-0x20 Unicode value.

Reply:
Sorry Terry, what needs to be changed here?

Reply:
There is no difference between teh two lines except white space. Whitespace was correct before; the hightlighted changes are after "An instance of" and after "SSL Certificates from". This is highly suspicious. 
![image](https://user-images.githubusercontent.com/19333103/157308492-746ea9da-008e-48d4-abc0-cc942ab92298.png)
Either the original was not a space (the byte 0x20), and as been changed to something that renders as a space; or the original *was* a space, and was changed to something that is not a space but renders as a space. Please use the Unix tool `xxd` to dump each version of the file and compare the hex differences to see what changed, and post the excerpt here.

Text that is not blank but renders as blank is a notorious way of creating security holes. We can't let that creep in, so we must resolve this now, and then change settings to prevent future occurrences (or sweep through all files to eliminate any remaining occurrences).

Reply:
![image](https://user-images.githubusercontent.com/18657477/157377898-ff8c3bce-51ad-40ad-83fb-d6a47e973a01.png)

Thanks for the explanation!. Please find the output of `xxd` command for the excerpt. it seems, the changes in the pull request are correct from the output.